src/index.js

```diff
@@ -0,0 +1,288 @@
+  },
+
+  init() {
+    this.elements.inp = document.createElement('textarea');
+    this.elements.main = document.createElement('div');
+    this.elements.keysContainer = document.createElement('div');
+    this.elements.inp.classList.add('result');
+    this.elements.main.classList.add('keyboard');
+    this.elements.keysContainer.classList.add('keyboard--keys');
+    const now = localStorage.getItem('language_Now') || 0;
+    this.elements.keysContainer.appendChild(this.createKeys(this.events.availableLanguages[now]));
+    this.elements.flag = false;
+    document.addEventListener('keyup', (e) => {
+      if (e.code === 'ControlLeft') this.elements.flag = true;
+      if (e.code === 'AltLeft' && this.elements.flag) {
+        this.elements.flag = false;
+        this.changeLang();
+        this.elements.keysContainer.appendChild(this.createKeys(this.events.languageNow));
+        window.location.reload();
+      }
+    });
+    document.addEventListener('keyup', (e) => {
+      if (e.code === 'ControlRight') this.elements.flag = true;
+      if (e.code === 'AltRight' && this.elements.flag) {
+        this.elements.flag = false;
+        this.changeLang();
+        this.elements.keysContainer.appendChild(this.createKeys(this.events.languageNow));
+        window.location.reload();
+      }
+    });
+    this.elements.keys = this.elements.keysContainer.querySelectorAll('.keyboard--key');
+    this.elements.main.appendChild(this.elements.keysContainer);
+    document.body.appendChild(this.elements.inp);
+    document.body.appendChild(this.elements.main);
+
+    document.addEventListener('keydown', (e) => {
+      document.querySelector(`.keyboard--key[data="${e.code}` || e.which || +'"]').classList.add('active');
+    });
+
+    document.addEventListener('keyup', (e) => {
+      document.querySelectorAll('.keyboard--key').forEach((element) => {
+        element.classList.remove('active');
+      });
+      e.preventDefault();
+      this.elements.inp.focus();
+      document.querySelector(`.keyboard--key[data="${e.code}` || e.which || +'"]').classList.remove('active');
+    });
+
+    document.addEventListener('mousedown', (e) => {
+      e.preventDefault();
+      this.elements.inp.focus();
+    });
+
+    document.querySelectorAll('.result').forEach((element) => {
+      element.addEventListener('focus', () => {
+        this.open(element.value, (currentValue) => {
+          const newElem = element;
+          newElem.value = currentValue;
+        });
+      });
+    });
+  },
+
+  createKeys(language) {
+    const fragment = document.createDocumentFragment();
+    language.forEach((key, i) => {
+      const keyElement = document.createElement('button');
+      const insertLineBreak = ['backspace', 'enter'].indexOf(key) !== -1;
+      keyElement.setAttribute('type', 'button');
+      keyElement.classList.add('keyboard--key');
+      keyElement.setAttribute('data', keyCodeLayout[i]);
+
+      switch (key) {
+        case 'backspace':
```

Comment:
usually in the switch case structure options are constants like BACKSPACE_KEY, TAB_KEY or BACKSPACE, TAB (as u wish) ... and they are declared at the beginning of the file